src/main/java/UserInterface.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Scanner;
+public class UserInterface {
+    Scanner sc;
+
+    public UserInterface(){
+        this.sc = new Scanner(System.in);
+    }
+
+    private void print(String message){
+        System.out.println(">>" + message + "\n");
+    }
+
+    protected void printWelcome(){
+        String logo = "\n" +
+                " ______       ___    _ .--.   .--.      .-''-.   \n" +
+                "|    _ `''. .'   |  | ||  | _/  /     .'_ _   \\  \n" +
+                "| _ | ) _  \\|   .'  | || (`' ) /     / ( ` )   ' \n" +
+                "|( ''_'  ) |.'  '_  | ||(_ ()_)     . (_ o _)  | \n" +
+                "| . (_) `. |'   ( \\.-.|| (_,_)   __ |  (_,_)___| \n" +
+                "|(_    ._) '' (`. _` /||  |\\ \\  |  |'  \\   .---. \n" +
+                "|  (_.\\.' / | (_ (_) _)|  | \\ `'   / \\  `-'    / \n" +
+                "|       .'   \\ /  . \\ /|  |  \\    /   \\       /  \n" +
+                "'-----'`      ``-'`-'' `--'   `'-'     `'-..-'   \n" +
+                "(◕ᴗ◕✿)\n";
+        System.out.println("Hewwo from" + logo);
+    }
+
+    protected void printAdd(Task task, int listSize){
```

Comment:
Out of curiosity, is there a reason for using `protected`?

Reply:
uh. I thought this was the best access modifier in this case. since it would be accessed by other classes but in the same package/folder. should it be something else?